Makefile

```diff
@@ -87,6 +87,13 @@ misspell:
 vet:
 	go vet $(MODULE)/...
 
+# Ensure that all log calls support contextual logging.
+test: logcheck
+.PHONY: logcheck
```

Comment:
Question: Should we run this in the GitHub actions?

Reply:
We aren't already running `test` for PRs?! We absolutely should.

I created https://github.com/kubernetes-sigs/dra-example-driver/issues/38 for E2E testing. Can you take care of doing something similar for unit testing?

Reply:
I have created #41